### ISSUE TITLE
SQL Syntax Exception for Oracle

### ISSUE THREAD

**Issue by kirenotneb** [ohdsi-studies/Covid19CharacterizationCharybdis#75]
Issue: SQLSyntaxErrorException `FROM` keyword not found where expected.

While running Charybdis it failed while running the "Creating time distributions" step due to a SQL Syntax Exception. Specifically the use of `size` as a column name while running the following code is not allowed as `SIZE` is apparently a reserved word in Oracle's dialect:

```
DBMS:
oracle

Error:
java.sql.SQLSyntaxErrorException: ORA-00923: FROM keyword not found where expected


SQL:
SELECT cohort_definition_id, COUNT(*) AS size FROM RDW_OMOP_RESULTS.AS_CHARYBDIS_RDW_OMOP_stg   WHERE cohort_definition_id IN (126,128,130,132,134,136,138,140,127,129,131,133,135,137,139,141) GROUP BY cohort_definition_id ;

R version:
R version 4.0.3 (2020-10-10)

Platform:
x86_64-apple-darwin19.6.0

Attached base packages:
- stats
- graphics
- grDevices
- utils
- datasets
- methods
- base

Other attached packages:
- Covid19CharacterizationCharybdis (1.2.0)
- DatabaseConnector (3.0.0)
```

@anthonysena, what steps should we take to mitigate this?

**Comment by anthonysena** [comment]
@kirenotneb - the time distribution feature is done via the OHDSI CohortDiagnostics package. Could you confirm what version of that package you have installed? I did a quick search in that package for this type of count query but did not see where it would alias the COUNT(*) with `size` as you reported.

**Comment by kirenotneb** [comment]
@anthonysena it's CohortDiagnostics_2.0.0. I installed Covid19CharacterizationCharybdis via `remotes::install_github`.

**Comment by schuemie** [comment]
Found it! It's in [FeatureExtraction](https://github.com/OHDSI/FeatureExtraction/blob/da48fda9f27cb3c8fa34cc3f91ee741071dc51bb/R/GetCovariates.R#L98)

**Comment by schuemie** [comment]
A bit odd, because that code passes our unit tests on our Oracle testing server. Anyway. I created the relevant [issue](https://github.com/OHDSI/FeatureExtraction/issues/106).

**Comment by anthonysena** [comment]
Thanks @schuemie for tracking that one down!

**Comment by kirenotneb** [comment]
I tested with the bugfix branch "issue-106-size-keyword" and it appears to have progressed beyond that initial error so I think we can consider this resolved once the fix is merged into the main line.
